### PR TITLE
fix(client): cancel RFQ button in non-graph view fixed (ARTP-961)

### PR DESCRIPTION
**Pull request by ramirozap**
Before: 
image

After: 
viewport width smaller than 320px: 
image
viewport width between 320px and 400px:
image
viewport width 400px and up:
image

### CODE REVIEW

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx [hunk 1]
```diff
@@ -22,6 +22,8 @@ const BookingPill = styled.div<{
   color: string
   isAnalyticsView: boolean
 }>`
+  padding-left: ${'1px'};
```
Comment (mozmorris) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: Does we need to use template placeholders`${someValue}` here?

**Comment by rikoe** [comment]
Can we have a screenshot of the change on the PR please?

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx (hunk 1 above)
Comment (ramirozap) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: Yes it is safe, I went with `padding-left: '1px'` as in a regular style object as first approach and that didn't work, thats why I used the templating afterwards.

**Comment by ramirozap** [comment]
> Can we have a screenshot of the change on the PR please?

Description Updated with screenshots

**Comment by mozmorris** [comment]
@ramirozap you'll need to rebase and fix the conflict in `TileBooking.tsx`. An `npm install` is needed after and then update the snapshots.

**Pull request merged**